### ISSUE TITLE
qBittorrent port failing after port change in gluetun.

### ISSUE THREAD

**Issue by holy-elbow** [soxfor/qbittorrent-natmap#19]
I am running this script with linuxserver qb and gluetun. My docker stack is below.

    version: "3.6"
    services:
      gluetun:
        image: qmcgaw/gluetun:latest
        container_name: gluetun
        cap_add:
            - NET_ADMIN
        devices:
            - /dev/net/tun:/dev/net/tun
        ports:
            - 8888:8888/tcp # HTTP proxy
            - 8388:8388/tcp # Shadowsocks
            - 8388:8388/udp # Shadowsocks
            - 8085:8085 # qbittorrent
        environment:
            # See https://github.com/qdm12/gluetun/wiki
            - VPN_SERVICE_PROVIDER=custom
            - VPN_TYPE=wireguard
            - VPN_ENDPOINT_IP=REDACTED
            - VPN_ENDPOINT_PORT=51820
            - WIREGUARD_PUBLIC_KEY=REDACTED
            - WIREGUARD_PRIVATE_KEY=REDACTED
            - WIREGUARD_ADDRESSES=10.2.0.2/32
            - VPN_PORT_FORWARDING=on
            - VPN_PORT_FORWARDING_PROVIDER=protonvpn
          # Edit this if you want to use OpenVPN:
          #- OPENVPN_USER=
          #- OPENVPN_PASSWORD=
          # Edit this if you want to use Wireguard:
          # Timezone for accurate log times
            - TZ=REDACTED
        restart: always
   
    qbittorrent:
      image: lscr.io/linuxserver/qbittorrent:latest
      container_name: qbittorrent
      network_mode: "service:gluetun"
      environment:
         - WEBUI_PORT=8085
         - PUID=1000
         - PGID=1000
      volumes:
         - /docker/qbittorrent/config:/config/
         - /data/downloads:/data/downloads/
      depends_on:
        gluetun:
          condition: service_healthy
      restart: unless-stopped

    qbittorrent_natmap:
      container_name: qbittorrent_natmap
      image: ghcr.io/soxfor/qbittorrent-natmap:latest
      environment:
         - QBITTORRENT_SERVER=10.2.0.2
         - VPN_GATEWAY=10.2.0.1
         - QBITTORRENT_PORT=8085
         - QBITTORRENT_USER=REDACTED
         - QBITTORRENT_PASS=REDACTED
      volumes:
         - /var/run/docker.sock:/var/run/docker.sock
      network_mode: "service:gluetun"
      depends_on:
        gluetun:
          condition: service_healthy
        qbittorrent:
          condition: service_started
      restart: unless-stopped
Not sure how to fix my issue as everytime gluetun becomes unhealthy the port changes. The script works and changes the port in qb, but the port is not connectable.

**Comment by Schmoaaaaah** [comment]
VPN_PORT_FORWARDING_PROVIDER=protonvpn is not in the example config and i think it conflicts with this setup as the container and gluetun both try to set the port.

**Comment by weareallricky** [comment]
@schmoaaaaah VPN_PORT_FORWARDING_PROVIDER=protonvpn is listed in the [gluetun readme for protonvpn](https://github.com/qdm12/gluetun-wiki/blob/main/setup/providers/protonvpn.md).

I'm curious if this is related (or same as) what I'm experiencing - #25 
@holy-elbow were you able to get this resolved?

**Comment by schmoaaaaah** [comment]
@weareallricky yes true, i actually meant that it is not present in the example of this project and that it is scruing up his povided configuration as both natmap and gluetun try to create a portforward. He either needs to remove VPN_PORT_FORWARDING_PROVIDER=protonvpn from his configuration and let natmap do the port request or keep it and let gluetun do the port request and replace natmap with something like this ghcr.io/geoffreycoulaud/glueforward. Since the port forwarding in gluetun for proton is working now i would recommend sticking with that and using glueforward.
